lib/improver/cli/nbhood.py

```diff
@@ -149,9 +143,6 @@ def process(cube: cli.inputcube,
     from improver.wind_calculations.wind_direction import WindDirection
 
     if neighbourhood_output == "percentiles":
-        if neighbourhood_shape == "square":
-            raise RuntimeError('neighbourhood_shape="square" cannot be used '
-                               'with neighbourhood_output="percentiles"')
```

Comment:
The corresponding `Raises` entries also need deleting from the doc-string.

Reply:
Done.